projects/rero/ng-core/src/lib/service/logger.service.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * RERO angular core
+ * Copyright (C) 2022 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Injectable } from '@angular/core';
+import { ILogger } from '../ILogger';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoggerService implements ILogger {
```

Comment:
Maybe rename as "ConsoleLoggerService" (in prevision ... maybe one day we will have FileLoggerService or TelepathyLoggerService")